docs/content/latest/manage/backup-restore/point-in-time-recovery.md

```diff
@@ -121,26 +121,19 @@ This feature is in active development. YSQL and YCQL support different features,
 
 ### YSQL limitations
 
-* Currently, you can only recover from the following YSQL operations:
+* The following list summarizes what's supported:
```

Comment:
I wonder if it even makes sense to have a list of what IS supported, in the limitations section though...

For eg, are there known ALTERs that DO NOT work, or more just ones we have not tested yet?

Reply:
Discussed offline.